apps/balancer-tools/src/lib/trimTrailingValues.test.ts

```diff
@@ -0,0 +1,71 @@
+import { trimTrailingValues } from "./utils";
```

Comment:
🤔 can you remind me why we needed this in the first place?

Reply:
It takes two arrays of numbers and a specified value to remove, and it returns new arrays with trailing occurrences of the specified value removed from both arrays. In that case I use to remove rounds where the pool didnt exist.

Reply:
on Pool Simulator we use this to remove the zones where the swap is not possible